### ISSUE TITLE
get_sleep_timestamps does not work with None as dates

### ISSUE THREAD

**Issue by dado93** [dado93/pylabfront#42]
If I run the following code:

```
loader = pylabfront.loader.LabfrontLoader(BASE_FOLDER)

start_date = None
end_date = None
user_id = ...

pylabfront.sleep.get_sleep_timestamps(loader, start_date, end_date, user_id)

```

I get the following error:

`TypeError: unsupported operand type(s) for -: 'NoneType' and 'datetime.timedelta'`

This is due to the fact that no check is done on the input parameters, even though None is set as default value for both start_date and end_date.

**Comment by dado93** [comment]
Related to #41 . The loading functions should accept multiple formats for the date parameter

**Comment by dado93** [comment]
Fixed with #45

**Comment by dado93** [comment]
This issue was closed with #45
